Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -285,6 +314,16 @@ void MediaPlayerPrivateGStreamerMSE::updateStates()
             GST_ERROR_OBJECT(pipeline(), "Setting the pipeline to PAUSED failed");
         m_isPipelinePlaying = false;
     }
+
+    if (m_isSeekPending) {
+        GST_DEBUG_OBJECT(pipeline(), "[Seek] committing pending seek to %s", toString(m_seekTime).utf8().data());
+        m_isSeekPending = false;
+        m_isSeeking = doSeek(m_seekTime, m_player->rate(), static_cast<GstSeekFlags>(GST_SEEK_FLAG_FLUSH));
```

Comment:
...if doSeek() is going to happen eventually and call seekToTime() eventually in line 219?

Reply:
Yes I agree if the seekToTime() is already called, we won't call it again in doSeek().
I will change this and i will test it.

Reply:
Is done.